onnx/reference/ops/op_max_pool.py

```diff
@@ -87,6 +87,10 @@ def _max_pool(  # type: ignore
                         + 1
                     )
                 )
+                if (output_spatial_shape[i] - 1) * strides[i] >= input_spatial_shape[
+                    i
+                ] + new_pads[i][0]:
```

Comment:
I would assign the condition to a variable and give it a meaningful name. Aka. make clear what this condition is checking.